src/com/etranslate/tm/processing/rtf/RTFParser.java

```diff
@@ -174,7 +174,7 @@ private static final String getJavaEncoding(int rtfCodepage) {
   // the current number of bytes to skip after a unicode character
   private Integer _currentSkipState = DEFAULT_SKIP_STATE;
   // a stack of skip states for bytes following a unicode character
-  private final Stack _ucSkipStates = new Stack();
+  private final Stack<Integer> _ucSkipStates = new Stack<>();
```

Comment:
I had a quick check last time, and I think the version of javacc needs to be bumped to 5.0 or later.
https://java.net/projects/javacc/lists/users/archive/2012-05/message/12